odl/discr/grid.py

```diff
@@ -681,9 +684,46 @@ def append(self, other):
         See Also
         --------
         insert
+        extend
         """
         return self.insert(self.ndim, other)
 
+    def extend(self, seq):
+        """Return a copy of ``self``, extended by all members of ``seq``.
+
+        Parameters
+        ----------
+        seq : sequence of `RectGrid`
```

Comment:
Second thought we should be really careful about this. Overall, we should do "what people expect", and for lists extend has the syntax `new_list = list.extend(something_that_is_like_a_list)`, implemented somewhat like this fella.

Anyway, wouldn't people expect this to work in a similar manner, e.g. `new_grid = grid.extend(something_that_is_like_a_grid)`, which would imply that calling this with a grid should work nicely. But I THINK that is not the case now, right?

We need to give this some more thought.